craco.config.js

```diff
@@ -126,12 +126,12 @@ module.exports = {
                             },
                             // Uniquify classnames and ids so that they are unique and
                             // don't conflict with each other
-                            {
+                            /* {
```

Comment:
This loader is being removed because I'm using global css classes to style one of the inner paths of an svg illustration (the `globe`). This loader would generate unique classes which I was unable to use from a css file.

A possible approach that would allow us to mantain this loader would be to access the path to style by using sibling selectors, but I personally prefer the current approach.

@satazor any thoughts on this?

Reply:
Even with this loader, you should be able to target them with selector prefixes. If that doesn’t work, you may always change the svg path to include attributes, such as data-xxx attributes so that you are able to target them.

Reply:
The approach chosen was to define the following option: `localIdentName: '[name]__[local]'`